### PR TITLE
Automatic updating of models

### PR DESCRIPTION
**Pull request by kasaby**
This removes the need to manually update models once they are updated in the service. It retrieves the set of models from the server and shows them in a suitable way.

### CODE REVIEW

File: src/views/selectmodel.js [hunk 1]
```diff
@@ -29,11 +32,11 @@ exports.initSelectModel = function(ctx) {
           $('<a>').attr('role', 'menu-item')
             .attr('href', '/')
             .attr('data-model', model.name)
-            .append(model.description)
+            .append(model.description.substring(0, model.description.length - 1), model.rate==8000?' (8KHz)':' (16KHz)'))
```
Comment (germanattanasio) [inline src/views/selectmodel.js]: do we only have 8 and 16k ?

File: src/models.js [hunk 2]
```diff
@@ -25,7 +24,24 @@ exports.getModels = function(options, callback) {
...
+    }
+    if( a.name < b.name) {
+      return -1;
+    }
+    return 0;
+    });
+    response.models=sorted;
+    localStorage.setItem('models', JSON.stringify(response.models));
+    var viewContext1 = {
+      currentModel: 'en-US_BroadbandModel',
```
Comment (germanattanasio) [inline src/models.js]: isn't  `en-US_BroadbandModel` supposed to be removed? @daniel-bolanos

Comment (daniel-bolanos) [inline src/models.js]: German, that is our main model, it is the default model for English, that one stays. (we only need to remove Japanese models and Michael's voice for TTS)

File: src/views/selectmodel.js (hunk 1 above)
Comment (daniel-bolanos) [inline src/views/selectmodel.js]: yes, that is right, broad and narrowband models for different languages. Sampling rates above 16k get converted to 16k at the service level.

File: src/models.js (hunk 2 above)
Comment (germanattanasio) [inline src/models.js]: My bad!

**Pull request merged**